tests/tx-decode-3.0.test.ts

```diff
@@ -46,8 +46,6 @@ test('stacks3.0 - decode tx - tenure change', () => {
       "previous_tenure_blocks": 1,
```

Comment:
I think you can just remove the trailing bytes from the serialized tx string above `0279be667ef9dcbbac55a06295ce870b07029bfcdb2dce28d959f2815b16f81798000000000000000000000000000000000000000000000000000000000000000000000000`

Right now it doesn't throw an error if there is extra bytes (something we should fix later)

Reply:
Done